tests/cli/test_output_option.py

```diff
@@ -17,97 +17,210 @@
 # limitations under the License.
 """Test behavior of ``--output`` option."""
 
+import pytest
 import yaml
 
 from renku.models.cwl import CWLClass
 
 
-def test_without_an_output_option(runner, client, run):
-    """Test detection of an output file without an output option."""
-    assert 0 == run(args=('run', 'touch', 'hello.txt'))
-    assert 1 == run(args=('run', 'touch', 'hello.txt'))
-    assert 0 == run(args=('run', '--no-output', 'touch', 'hello.txt'))
+@pytest.fixture
+def cli(client, run):
```

Comment:
good idea to make this fixture! But why not put it with the others in confest.py?